assets/js/phoenix_live_view.js

```diff
@@ -406,8 +406,8 @@ export class LiveSocket {
       let target = closestPhxBinding(e.target, click)
       let phxEvent = target && target.getAttribute(click)
       if(!phxEvent){ return }
-      e.preventDefault()
-     
+      e.stopPropagation()
```

Comment:
I think the other ones we want `preventDefault` b/c we are clicking links and submitting forms and we take over.  With this case we either want `stopPropagation` or nothing.  Lmk what we should do!